### ISSUE TITLE
[Bug] Narration is missing for "Toby Accessibility Mod Options" menu

### ISSUE THREAD

**Issue by proydoha** [Alando1-doom/Toby-Accessibility-Mod-for-Doom#180]

**Comment by Alando1-doom** [comment]
I can get to work on the voiceovers for this menu section this weekend.

**Comment by proydoha** [comment]
I think this was fixed at some point. I've clicked through "Toby Accessibility Mod Options" menu and all menu items seem to have narration.
